ROX-12391: Revisit public docs about Central/Sensor compatibility (#52881)
openshift/openshift-docs: upgrading/upgrade-helm.adoc

* Update configuration files for the secured-cluster-services Helm chart.
* Upgrade the secured-cluster-services Helm chart.

[IMPORTANT]
====

> * In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.

Reviewer: I suggest the following rewording
```suggestion
* In order to ensure optimal functionality it is recommended to keep your secured-cluster-services on the same version as central-services.
```

Author: 👍

> * Enabling automated upgrades for your secured clusters is strongly recommended. Should you prefer manually upgrading your secured clusters refer to the xref:upgrade-roxctl.adoc#upgrade-secured-clusters[manual on upgrading secured clusters].

Reviewer: I don't think that there are any automatic upgrades for Helm that are provided by ACS itself. This is something customers should arrange with their GitOps tools. I suggest removing this point.

```suggestion
```

Author: Fair, I'll change it.

But is there a reason why we don't support automatic upgrades using helm charts? Sounds like something we should do if possible.

[IMPORTANT]
====

> * In order to ensure optimal functionality it is recommended to keep your secured-cluster-services on the same version as central-services.

Reviewer: Probably this would look better without the bulleted list decoration because there's just one item.

```suggestion
In order to ensure optimal functionality it is recommended to keep your secured-cluster-services on the same version as central-services.
```

Author: Fair point, done
